website/pages/en/network/developing.mdx

```diff
@@ -2,52 +2,122 @@
+Learn developer fundamentals and review the basics of a subgraph’s lifecycle. To start coding right away and engage The Graph, go to the following: [Developer Quick Start](/quick-start/)
 
-## Subgraph Lifecycle
+## Overview
 
-Subgraphs deployed to the network have a defined lifecycle.
+As a developer, you need important data to build and power your dapps. Querying and indexing that blockchain data can be challenging, but The Graph provides a solution to this issue.
 
-### Build locally
+On The Graph, you can:
 
-As with all subgraph development, it starts with local development and testing. Developers can use the same local setup whether they are building for The Graph Network or a local Graph Node, leveraging `graph-cli` and `graph-ts` to build their subgraph. Developers are encouraged to use tools such as [Matchstick](https://github.com/LimeChain/matchstick) for unit testing to improve the robustness of their subgraphs.
+1.  Create, deploy and publish subgraphs to The Graph using **Graph CLI** and [Subgraph Studio](/https://thegraph.com/studio/).
+2.  Use Graph node via **GraphQL** to query live subgraphs and power dapps.
 
-> There are certain constraints on The Graph Network, in terms of feature and network support. Only subgraphs on [supported networks](/developing/supported-networks) will earn indexing rewards, and subgraphs which fetch data from IPFS are also not eligible.
+### What is GraphQL and Graph CLI?
 
-### Deploy to Subgraph Studio
+- **GraphCLI** is a command line interface tool for building and deploying to The Graph.
+- **GraphQL** is the query language use for APIs and a runtime for executing those queries with your existing data. The Graph uses GraphQL to query subgraphs.
 
-Once defined, the subgraph can be built and deployed to [Subgraph Studio](/deploying/subgraph-studio-faqs/). Subgraph Studio is a sandbox environment which will index the deployed subgraph and make it available for rate-limited development and testing. This gives developers an opportunity to verify that their subgraph does not encounter any indexing errors, and works as expected.
+### Developer Capabilities
 
-### Publish to the Network
+- Engage with the entire subgraph lifecycle on the network, including publishing, querying and iterating on your custom subgraphs.
+- Specialize in subgraph development, building open APIs for others to utilize and build on.
+- Focus on application development by querying subgraphs deployed by other developers.
 
-When the developer is happy with their subgraph, they can publish it to The Graph Network. This is an on-chain action, which registers the subgraph so that it is discoverable by Indexers. Published subgraphs have a corresponding NFT, which is then easily transferable. The published subgraph has associated metadata, which provides other network participants with useful context and information.
+### Network Economics
```

Comment:
imo network economics should be at the end of this page and move the subgraph specifics and lifecycle up here. 

Developers are the persona here and more important for them is to first build the subgraph out. The network economics is once they are deployed to the network.